website/client/src/components/shops/balanceInfo.vue

```diff
@@ -67,33 +71,40 @@ export default {
   computed: {
     currencies () {
       const currencies = [];
-      currencies.push({
-        type: 'hourglasses',
-        icon: this.icons.hourglasses,
-        value: this.userHourglasses,
-      });
+      if (this.neededCurrencyOnly === true) {
+        currencies.push({
+          type: 'gems',
```

Comment:
This should not be hard-coded. We might want to display Hourglasses only, or Gold only, in other places.

Reply:
Yep, I know. Need to figure it out!